hw/top_earlgrey/ip/clkmgr/data/autogen/clkmgr_sec_cm_testplan.hjson

```diff
@@ -25,166 +25,81 @@
   testpoints: [
     {
       name: sec_cm_bus_integrity
-      desc: '''Verify the countermeasure(s) BUS.INTEGRITY.
-            This entry is covered by tl_access_test.
-            '''
+      desc: "Verify the countermeasure(s) BUS.INTEGRITY."
       stage: V2S
-      tests: ["clkmgr_tl_intg_err"]
+      tests: []
```

Comment:
Why is this file still needed? it seems redundant, given that we have the one within `hw/ip_templates` that gets rendered into `hw/top_earlgrey/ip_autogen`...

Reply:
Oops! Apparently I forgot to delete the tree under hw/top_earlgrey/ip/clkmgr in the commit before last. Done now.